app/views/oralReport/page6.scala.html

```diff
@@ -11,93 +11,52 @@
+                @checkboxDetailsField(reportForm, "issueEmployment", "Employment, training and education", "Provide a brief assessment of employment, training and education")
+                @checkboxDetailsField(reportForm, "issueFinance", "Finances", "Provide a brief assessment of finances")
+                @checkboxDetailsField(reportForm, "issueRelationships", "Relationships", "Provide a brief assessment of relationships")
+                @checkboxDetailsField(reportForm, "issueSubstanceMisuse", "Substance misuse", "Provide a brief assessment of substance misuse")
+                @checkboxDetailsField(reportForm, "issueHealth", "Physical & mental health", "Provide a brief assessment of physical & mental health")
+                @checkboxDetailsField(reportForm, "issueBehaviour", "Thinking & behaviour", "Provide a brief assessment of thinking & behaviour")
+                @checkboxDetailsField(reportForm, "issueMaturity", "Maturity", "Provide a brief assessment of maturity")
                 @checkboxDetailsField(reportForm, "issueOther", "Other (Please specify below)", "Provide a brief assessment of other issues")
             </div>
         </fieldset>
     </div>
 
-        <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
-
-        <div class="govuk-form-group" data-module="radios">
-            <fieldset class="govuk-fieldset">
-                <legend class="govuk-fieldset__legend govuk-label">
-                    Is there evidence of the offender experiencing trauma?
-                </legend>
-
-                @inputRadioGroup(
-                    reportForm("experienceTrauma"),
-                    options(
-                        "yes" -> "Yes",
-                        "no" -> "No"
-                    ),
-                    Symbol("_inline") -> "true"
-                )
-            </fieldset>
-        </div>
-
-        <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
-
-        <div class="govuk-form-group" data-module="radios">
-            <fieldset class="govuk-fieldset">
-                <legend class="govuk-fieldset__legend govuk-label">
-                    Does the offender have caring responsibilities for children or adults, or have they ever had caring responsibilities for children or adults?
-                </legend>
-
-                @inputRadioGroup(
-                    reportForm("caringResponsibilities"),
-                    options(
-                        "yes" -> "Yes",
-                        "no" -> "No"
-                    ),
-                    Symbol("_inline") -> "true"
-                )
-            </fieldset>
-        </div>
-
-        <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
-
-        @textarea(reportForm("assessmentEvidence"), '_label -> "Evidence for assessment")
```

Comment:
Does this also need removing from OralReportData and the hidden inputs file?

Reply:
I thought I'd removed it but I'll have a look